pkg/workceptor/remote_work.go

```diff
@@ -531,7 +531,11 @@ func (rw *remoteUnit) Start() error {
 
 // Restart resumes monitoring a job after a Receptor restart
 func (rw *remoteUnit) Restart() error {
-	return rw.startOrRestart(false)
+	red := rw.Status().ExtraData.(*remoteExtraData)
+	if red.RemoteStarted {
+		return rw.startOrRestart(false)
+	}
+	return fmt.Errorf("remote work had not previously started")
```

Comment:
Should this be an error?  I was thinking work in this state would just be silently discarded.

Reply:
returning an error here results in a Warning in caller https://github.com/project-receptor/receptor/blob/0374f78d160383fbc92a1a2bc28a1c1a319729a4/pkg/workceptor/workceptor.go#L227

are you suggesting we do a `release` in this scenario?

Reply:
A warning sounds fine.  I think my concern here was not to keep retrying.  Do we mark the unit as failed?

I don't think we should do a release until the user actually asks for one.

Reply:
Yep, unit is marked as failed